glossary/Argument.md

```diff
@@ -0,0 +1,3 @@
+### Argument
+
+An argument is a value passed as an input to a function and can be either a primitve or an object.
```

Comment:
You may also add here that in JS, you can pass a function as function argument as well as functions are first-class objects in JavaScript

Reply:
Definitely worth stating here, I'll update the term.

Reply:
I'd put it as a note like
> ... or an object(a function is an object)

or some other way

Reply:
Hmmm, I could rephrase the first sentence to be something along the lines of *An argument is a value passed as an input to a function and can be either a primitve, a function or an object.* Seems less weird that way.